### ISSUE TITLE
sudo not really implemented

### ISSUE THREAD

**Issue by cleoo** [PawarPawan/arpscan-new#1]
Hi,

Command built with suspawn is like :
sudo -- sh -c "arp-scan -l ..."

Rules added in sudoers file, to avoid typing password, can't obviously be effective
my_user ALL=(ALL) NOPASSWD: /usr/bin/arp-scan

Method using chmod u+s /usr/bin/arp-scan, is not "upgrade compliant"
Granting sh with NOPASSWD is totallly unsafe

Do you have a workaround ?

Regards
F.

**Comment by PawarPawan** [comment]
Yes I have worked on that too, but the result was not updating until the arp table of the system not updated you can check it in lib/arpscanner.old.js file. As you know system arp not updated immediately as a new device inserted into the network. Need to find other solution. Have you any other idea?
